tests/conftest.py

```diff
@@ -1367,8 +1367,13 @@ def cluster_teardown_finalizer():
             # If KMS is configured, clean up the backend resources
             # we are doing it before OCP cleanup
             if config.DEPLOYMENT.get("kms_deployment"):
-                kms = KMS.get_kms_deployment()
-                kms.cleanup()
+                try:
+                    kms = KMS.get_kms_deployment()
+                    kms.cleanup()
+                except Exception as ex:
+                    log.error(f"Failed to cleanup KMS. Exception is: {ex}")
```

Comment:
One downside of this is that we will never get this info if it will happen as we will continue the job and think that it was successful and we will never get this fixed if it was not temporary issue.

@shylesh any input on this? 
@vavuthu is this happening permanently? If so we will need to fix the root cause.
If it was just temp issue then I think we should probably continue failing the job to teardown as the cleanup is required in vault side IIUC and if it's not done here we have leftovers.

I would wait with @shylesh comment here before merging.

Reply:
@petr-balogh correct, vault will have stale entries.  Earlier you suggested me to do the cleanup script directl y on the backend side just like aws cleanup job, I looked at it for couple of times but since it involves combing jenkins for alive cluster and comparing to the entries in vault I couldn't comeup with any simple implementation and it's been a backlog. For now we can merge this and I will create a ticket for this  and might have to work if priority is high.

Reply:
@petr-balogh teardown failing whenever we have issue with KMS ( currently for KMS, certs are expired )